glib/src/object.rs

```diff
@@ -3376,6 +3401,65 @@ wrapper! {
     }
 }
 
+type WeakRefNotifyData = ManuallyDrop<Pin<Box<Option<Box<dyn FnOnce() + 'static>>>>>;
```

Comment:
Why the `Pin` and why the double boxing and why the `Option` and why the `ManuallyDrop`? This at least needs some comment explaining this contraption :)